src/Features/CSharp/Portable/CodeRefactorings/SyncNamespace/CSharpChangeNamespaceService.cs

```diff
@@ -358,8 +358,7 @@ protected override async Task<SyntaxNode> TryGetApplicableContainerFromSpanAsync
             }
 
             return container;
-
-            bool ContainsNamespaceDeclaration(SyntaxNode node)
+            static bool ContainsNamespaceDeclaration(SyntaxNode node)
```

Comment:
>ContainsNamespaceDeclaration [](start = 24, length = 28)

It's probably good to keep the empty line above to separate the local function.

Reply:
hrrmm... this def seems to be a defect in the fixer.  It should not be removing that blank line.

Reply:
Is this tracked somewhere? I believe I experience fixes removing empty lines or final line breaks quite often.